build_deploy.sh

```diff
@@ -15,19 +15,25 @@ export CONTAINER_BUILD_EXTRA_PARAMS=${CONTAINER_BUILD_EXTRA_PARAMS:-"--no-cache"
 export PROW_JOBS_SCRAPER_IMAGE=${PROW_JOBS_SCRAPER_IMAGE:-"quay.io/app-sre/prow-jobs-scraper"}
 
 # Tag with the current commit sha
-PROW_JOBS_SCRAPER_TAG="$(git rev-parse --short=7 HEAD)"
+PROW_JOBS_SCRAPER_TAG="$(git rev-parse HEAD)"
 export PROW_JOBS_SCRAPER_TAG
 
 # Setup credentials to image registry
-${CONTAINER_CMD} login -u="${QUAY_USER}" -p="${QUAY_TOKEN}" quay.io
+#${CONTAINER_CMD} login -u="${QUAY_USER}" -p="${QUAY_TOKEN}" quay.io
 
 # Build and push latest image
 make build-image
 
 PROW_JOBS_SCRAPER_IMAGE_COMMIT_SHA="${PROW_JOBS_SCRAPER_IMAGE}:${PROW_JOBS_SCRAPER_TAG}"
-${CONTAINER_CMD} push "${PROW_JOBS_SCRAPER_IMAGE_COMMIT_SHA}"
+echo ${CONTAINER_CMD} push "${PROW_JOBS_SCRAPER_IMAGE_COMMIT_SHA}"
```

Comment:
the echo will just print, and it won't actually not push the tag

Reply:
my bad, I wanted to check that the images look fine and forget to remove it